### PR TITLE
DC-631 Next Core package release - Changes in the AsciiDoc side.

### PR DESCRIPTION
**Pull request by eltesoro-koji**

### CODE REVIEW

**Review by rasienko — APPROVED** [review]
Looks really good! Couple of tiny things where I noticed some differences between package and static docs. You can fix any that make sense or just merge as is.

File: src/asciidoc/reference/core/coreFrontendOverview.adoc [hunk 1]
```diff
@@ -84,29 +84,29 @@ const backendURL = Koji.services.backend
...
+* _<<core-frontend-ui-upload#,Upload>>_, Class that uploads files from the frontend of your Koji app directly to your project's CDN.
 
 == Methods
 
 [.hcode, id=".config", reftext="config"]
 === .config(kojiConfig, kojiConfigOptions)
 
 Initializes this package for use with configuration data from the `koji.json` file.
-This method sets up the services, remix values, development environment, and deployment instructions for your Koji template.
+This method sets up the services, configuration values, development environment, and deployment instructions for your Koji app.
```
[the remark this answers is not in the document]
Comment (eltesoro-koji) [reply src/asciidoc/reference/core/coreFrontendOverview.adoc:101]: Replaced 'configuration' with 'customization'.

File: src/asciidoc/reference/core/coreFrontendOverview.adoc [hunk 2]
```diff
@@ -33,17 +33,17 @@ Defaults to `false`.
...
+_<<core-frontend-iap#,IAP>>_, Class that manages in-app purchase transactions on the frontend of your Koji app.
 
 [.hcode, id=".identity", reftext="identity"]
 === .identity
 
-_<<core-frontend-identity#,Identity>>_, Class that manages authentication and authorization on the frontend of your Koji template.
+_<<core-frontend-identity#,Identity>>_, Class that manages authentication and authorization on the frontend of your Koji app.
 
 [.hcode, id=".isReady", reftext="isReady"]
 === .isReady
```
[the remark this answers is not in the document]
Comment (eltesoro-koji) [reply src/asciidoc/reference/core/coreFrontendOverview.adoc:49]: @rasienko: I don't understand this comment. It doesn't seem to match the highlighted area. It's a class, not a boolean.

File: src/asciidoc/docs/_includes/interface-KojiConfig.adoc [hunk 3]
```diff
@@ -8,11 +8,11 @@ Configuration data for the Koji.
 
 ==== Properties
 
-* `@@initialTransform` – _Any_, (Optional) Placeholder values for new remixes.
+* `@@initialTransform` – _Any_, (Optional) Placeholder values for new customized versions.
 * `deploy` – _Any_, (Optional) Instructions for deploying the services to production.
 * `develop` – _Any_, (Optional) Instructions for setting up the services in a development/editor environment.
 * `metadata` – _<<KojiMetadata>>_, (Optional) Metadata about the project and creator.
-* `remixData` – _Any_, (Optional) Default values for the customizable remix data.
+* `remixData` – _Any_, (Optional) Default values for the customizable data.
```
[the remark this answers is not in the document]
Comment (eltesoro-koji) [reply src/asciidoc/docs/_includes/interface-KojiConfig.adoc:15]: Replaced 'customizable data' with 'configuration data' for consistency. Will set both places back to 'customizable' next time, but low priority.

File: src/asciidoc/docs/_includes/backend-basic-use.adoc [hunk 4]
```diff
@@ -6,7 +6,7 @@ Import the package in your backend code.
 [source,javascript]
 import { KojiBackend } from '@withkoji/core';
 
-Initialize the package with your configuration data, and use `KojiBackend.middleware` to scope operations per remix.
+Initialize the package with your configuration data, and use `KojiBackend.middleware` to scope operations per customized version.
```
[the remark this answers is not in the document]
Comment (eltesoro-koji) [reply src/asciidoc/docs/_includes/backend-basic-use.adoc:9]: Fixed.

File: src/asciidoc/reference/core/coreFrontendOverview.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (rasienko) [reply src/asciidoc/reference/core/coreFrontendOverview.adoc:49]: Sorry, that was for the isReady property description. I couldn't put it on the exact line so I put it close, but I should have mentioned that.

[the remark this answers is not in the document]
Comment (eltesoro-koji) [reply src/asciidoc/reference/core/coreFrontendOverview.adoc:49]: Fixed.

**Review by rasienko — APPROVED** [review]
LGTM!

**Pull request merged**